src/net/sourceforge/kolmafia/AdventureResult.java

```diff
@@ -1475,4 +1472,18 @@ public static WildcardResult getInstance(String text) {
       return null;
     }
   }
+
+  private int safeDowncastLongToInt(long value) {
```

Comment:
Should this be in a utility class elsewhere? Can you add tests for it?

Reply:
But for some reason I didn't see that V has already questioned the necessity of the change in the first place.

Reply:
> Should this be in a utility class elsewhere? Can you add tests for it?

Yes.  But the equivalent is Ints.saturatedCast in Google's Guava and I wasn't sure I wanted to pull in the entire library for one routine.  But at least the routine would have been tested :-)